java/spacewalk-java.spec

```diff
@@ -172,22 +173,26 @@ Requires:       c3p0 >= 0.9.1
 Requires:       cglib
 %if 0%{?suse_version}
 Requires:       classmate
-Requires:       javamail
 %endif
 Requires:       cobbler >= 3.0.0
 Requires:       concurrent
 Requires:       dwr >= 3
 Requires:       %{ehcache}
-Requires:       gnu-jaf
+Requires:       (jaf or gnu-jaf)
 Requires:       google-gson >= 2.2.4
 Requires:       hibernate-commons-annotations
 Requires:       hibernate5
 Requires:       httpcomponents-client
 Requires:       ical4j
 Requires:       jade4j
 Requires:       %{apache_commons_digester}
+%if 0%{?rhel} >= 8
+BuildRequires:  java-11-openjdk
```

Comment:
This BuildRequires exists already above. Maybe this should be just a "Require" ?

Reply:
Wer kopiert, der verliert...